src/recipes/heartbeat-by-user-first-impression/config.js

```diff
@@ -47,41 +46,98 @@ const filterFields = [
 ];
 
 
+
+
 const engagementRules = [
   {
-    alias: '^en, release',
+    alias: '^cs, Aurora',
     rule: {
-      locale: /^en/i,
-      updateChannel: /^release/i
+      locale: /^cs/i,
+      updateChannel: /^aurora/i
     },
     urls: [
+      "https://qsurvey.mozilla.com/s3/Developer-Tools-Language"
```

Comment:
should be:  (because below):

"https://qsurvey.mozilla.com/s3/Developer-Tools-Language?locale=cs" 

or such.